src/pages/communication-page/__tests__/communication.spec.ts

```diff
@@ -109,6 +116,17 @@ describe('CommunicationPage', () => {
       expect(component).toBeDefined();
     });
 
+    describe('DOM', () => {
+      it('should display the correct address fields', () => {
```

Comment:
To make sure every possibility is covered, splitting this into 2 tests one which tests everything being there and one which tests everything missing.

Reply:
fixed